peekingduck/utils/mapping_efficientdet.txt

```diff
@@ -0,0 +1,82 @@
+{
+    "person": 0,
+    "bicycle": 1,
+    "car": 2,
+    "motorcycle": 3,
+    "aeroplane": 4,
+    "bus": 5,
+    "train": 6,
+    "truck": 7,
+    "boat": 8,
+    "traffic light": 9,
+    "fire hydrant": 10,
+    "stop sign": 12,
+    "parking meter": 13,
+    "bench": 14,
+    "bird": 15,
+    "cat": 16,
+    "dog": 17,
+    "horse": 18,
+    "sheep": 19,
+    "cow": 20,
+    "elephant": 21,
+    "bear": 22,
+    "zebra": 23,
+    "giraffe": 24,
+    "backpack": 26,
+    "umbrella": 27,
+    "handbag": 30,
+    "tie": 31,
+    "suitcase": 32,
+    "frisbee": 33,
+    "skis": 34,
+    "snowboard": 35,
+    "sports ball": 36,
+    "kite": 37,
+    "baseball bat": 38,
+    "baseball glove": 39,
+    "skateboard": 40,
+    "surfboard": 41,
+    "tennis racket": 42,
+    "bottle": 43,
+    "wine glass": 45,
+    "cup": 46,
+    "fork": 47,
+    "knife": 48,
+    "spoon": 49,
+    "bowl": 50,
+    "banana": 51,
+    "apple": 52,
+    "sandwich": 53,
+    "orange": 54,
+    "broccoli": 55,
+    "carrot": 56,
+    "hot dog": 57,
+    "pizza": 58,
+    "donut": 59,
+    "cake": 60,
+    "chair": 61,
+    "couch": 62,
+    "potted plant": 63,
+    "bed": 64,
+    "dining table": 66,
+    "toilet": 69,
+    "tv": 71,
+    "laptop": 72,
+    "mouse": 73,
+    "remote": 74,
+    "keyboard": 75,
+    "cell phone": 76,
+    "microwave": 77,
+    "oven": 78,
+    "toaster": 79,
+    "sink": 80,
+    "refrigerator": 81,
+    "book": 83,
+    "clock": 84,
+    "vase": 85,
+    "scissors": 86,
+    "teddy bear": 87,
+    "hair drier": 88,
+    "toothbrush": 89,
+}
```

Comment:
Would it be better for these mapping files to be in their respective model directories instead of `utils`? For consideration when @liyier90 reviews this PR.

Reply:
TBH, I'm not 100% sure yet which is a good location for these files, based on the following considerations:

1. Store mapping files with models
- Cleaner organisation (+)
- Same (duplicate) mappings for multiple models, e.g. Yolov4 and YoloX (-)

2. Store mapping files in `utils` (or somewhere else)
- One file for multiple models, no duplication (c.f. above) (+)
- Cleaner implementation code for reading mapping files (+?)
- File organisation not as clean (-)

Welcome anyone to chip in to this discussion.

Reply:
I'm leaning towards not keeping them with the individual models to avoid having duplicated files.

To mirror my comment below, I'm also wondering if it's better to categorize the mapping as maybe "coco original numbering" (efficientdet) vs "coco condensed" (yolo, yolox). "coco condensed" is basically when they "shift the IDs up" so there's no skipped numbers.

Reply:
Refactored this into a single master mapping file.